tests/unit/test_charm_status.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+import unittest
+from unittest.mock import patch
+from helpers import patch_network_get
+import hypothesis.strategies as st
+from hypothesis import given
+from ops.model import ActiveStatus, BlockedStatus
+from ops.testing import Harness
+from charm import PrometheusCharm
+
+logger = logging.getLogger(__name__)
+
+
+class TestActiveStatus(unittest.TestCase):
+    """Feature: Charm's status should reflect the correctness of the config / relations.
+
+    Background: When launched on its own, the charm should always end up with active status.
+    In some cases (e.g. Ingress conflicts) the charm should go into blocked state.
+    """
+
+    def setUp(self) -> None:
+        self.app_name = "prometheus-k8s"
+
+    @patch_network_get(private_address="1.1.1.1")
+    @patch("charm.KubernetesServicePatch", lambda x, y: None)
+    @given(st.booleans(), st.integers(1, 5))
+    def test_unit_is_active_if_deployed_without_config_or_relations(self, is_leader, num_units):
```

Comment:
- Should a Prometheus charm block if it has no relations ?
- The Prometheus charm always sets a default config which makes it scrape itself.

In view of the above two concerns it is not clear to me what this method (test case) is testing for ? In particular what is new about this test case compared to the ones that already exist.